### DISCUSSION TITLE
Forums

### DISCUSSION THREAD

**Discussion by Redder358** [Tyrrrz/DiscordChatExporter#1007] [Q&A]
Hi, I want to use DiscordCHatExporter to make a copy of a forums chat. However, it doesn't see different forums "subchannels" as in discord, it sees them all as a single empty channel. Is there a way to make a copy of forums chat using your or some other program?

**Comment by Tyrrrz** [comment]
Forum "channels" are essentially threads, which are only partially supported right now. #999
